src/chainparams.cpp

```diff
@@ -322,6 +334,9 @@ class CRegTestParams : public CChainParams {
         assert(consensus.hashGenesisBlock == uint256S("0x0f9188f13cb7b2c71f2a335e3a4fc328bf5beb436012afca590b1a11466e2206"));
         assert(genesis.hashMerkleRoot == uint256S("0x4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b"));
 
+        // All non-zero coinbase outputs must go to this scriptPubKey
+        consensus.mandatory_coinbase_destination = StrHexToScriptWithDefault(gArgs.GetArg("-con_mandatorycoinbase", ""), CScript()); // Blank script allows any coinbase destination
```

Comment:
Why not just 
```suggestion
        // Blank script allows any coinbase destination
        consensus.mandatory_coinbase_destination = ParseHex(args.GetArg("-con_mandatorycoinbase", ""));
```

Reply:
Does it implicitly convert it? It's a script, not a series of bytes.

Reply:
It seems to be working on https://github.com/ElementsProject/elements/pull/433/commits/f0391db8e83dc8ab320f0001795d58fd201b7c97#diff-64cbe1ad5465e13bc59ee8bb6f3de2e7R431

Reply:
Hm ok in that case we don't need StrHexToScriptWithDefault at all, though I'll make the conversion explicit.

Reply:
Oh right C++: It can implicitly use constructor.